asm/instruction.go

```diff
@@ -527,6 +527,69 @@ func (insns Instructions) Tag(bo binary.ByteOrder) (string, error) {
 	return hex.EncodeToString(h.Sum(nil)[:unix.BPF_TAG_SIZE]), nil
 }
 
+// RewriteJumps updates all jumps to also take the symbol offset into account.
```

Comment:
This doesn't explain that map references are also taken care of. I think calling this `FixupReferences` would be better. We already have other `Rewrite` functions which allow the user to specify what to rewrite with, which isn't true here.

Reply:
renamed it to `FixupReferences` :+1: